app/src/main/java/com/safetyheads/akademiaandroida/AndroidAcademyApplication.kt

```diff
@@ -28,9 +30,10 @@ class AndroidAcademyApplication: Application() {
         //usecases
         single<SplashScreenUseCase> { SplashScreenUseCaseImpl() }
         single { LoadItemsToDropDownListUseCase() }
+        single { GetConfigUseCase(ConfigRepositoryUseCaseImpl()) }
```

Comment:
Please add separated definition for ConfigRepositoryUseCaseImpl maybe in future would need `ConfigurationRepository"